File: Lab 3/src/Page.java
import java.util.Random;

public class Page {
    public String name;
    int memory_size;
    public int time;
    int appeal;
    boolean flag = true;

    public Page(String name, int memory_size, int time) {
        this.name = name;
        this.memory_size = memory_size;
        this.time = time;
        Random rnd = new Random();
        this.appeal = rnd.nextInt(20);
    }
}

File: 3 semak/Other/3 ab/Lab3/Lab3/Page.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab3
{
    internal class Page
    {
        public String name;
        public int memory_size;
        public int time;
        public bool flag = true;

        public Page(String name, int memory_size, int time)
        {
            this.name = name;
            this.memory_size = memory_size;
            this.time = time;
        }
    }
}
